src/app/code/community/Shopgate/Cloudapi/Helper/Request.php

```diff
@@ -38,19 +48,19 @@ class Shopgate_Cloudapi_Helper_Request extends Mage_Core_Helper_Abstract
      */
     public function isShopgateRequest()
     {
-        if ($this->parameterDetected()) {
+        if ($this->parameterInAppDetected()) {
             $this->setCookie();
 
             return true;
         }
 
-        return $this->cookieIsSet();
+        return $this->cookieIsSet(self::KEY_SGCLOUD_INAPP,  self::COOKIE_VALUE);
```

Comment:
space

Reply:
Still one additional space, please remove it.